presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedSmokeTestBase.java

```diff
@@ -484,33 +488,83 @@ private void testCreateTableLike()
-                "   format = 'PARQUET'\n" +
+                "   format = 'PARQUET',\n" +
+                "   format_version = '1'\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy1");
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy2 (LIKE test_create_table_like_original EXCLUDING PROPERTIES)");
         assertEquals(getTablePropertiesString("test_create_table_like_copy2"), "WITH (\n" +
-                "   format = 'PARQUET'\n" +
+                "   format = 'PARQUET',\n" +
+                "   format_version = '1'\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy2");
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy3 (LIKE test_create_table_like_original INCLUDING PROPERTIES)");
         assertEquals(getTablePropertiesString("test_create_table_like_copy3"), "WITH (\n" +
                 "   format = 'PARQUET',\n" +
+                "   format_version = '1',\n" +
                 "   partitioning = ARRAY['adate']\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy3");
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy4 (LIKE test_create_table_like_original INCLUDING PROPERTIES) WITH (format = 'ORC')");
         assertEquals(getTablePropertiesString("test_create_table_like_copy4"), "WITH (\n" +
                 "   format = 'ORC',\n" +
+                "   format_version = '1',\n" +
                 "   partitioning = ARRAY['adate']\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy4");
 
         dropTable(session, "test_create_table_like_original");
     }
 
+    @Test
+    public void testCreateTableWithFormatVersion()
+    {
+        testWithAllFormatVersions(this::testCreateTableWithFormatVersion);
+    }
+
+    private void testCreateTableWithFormatVersion(Session session, String formatVersion)
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_create_table_with_format_version " +
+                "WITH (" +
+                "format = 'PARQUET', " +
+                "format_version = '" + formatVersion + "'" +
+                ") " +
+                "AS " +
+                "SELECT orderkey AS order_key, shippriority AS ship_priority, orderstatus AS order_status " +
+                "FROM tpch.tiny.orders";
+
+        assertUpdate(session, createTable, "SELECT count(*) from orders");
+
+        String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.%s (\n" +
+                        "   \"order_key\" bigint,\n" +
+                        "   \"ship_priority\" integer,\n" +
+                        "   \"order_status\" varchar\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'PARQUET',\n" +
+                        "   format_version = '" + formatVersion + "'\n" +
```

Comment:
I know you're referencing `testCreatePartitionedTableAs`, but the formatting doesn't look good in IntelliJ. This statement uses `format()` but for format_version, it's using `+`. Would you please use a unified way? I think the createTable stmt looks better. Also I don't know why this string uses "\n" while the above one doesn't.

Reply:
I have made the change and used format for all the strings. 

> Also I don't know why this string uses "\n" while the above one doesn't.

The first string is SQL query to run to create the test table and this string is the output for SHOW CREATE TABLE statement in Presto. This is how presto prints the output so to assert the output with expected output we need to add `/n` in the string.